deploy/test/config/k8s/helm-app.yaml

```diff
@@ -0,0 +1,29 @@
+# This app is created to run end-to-end with the Secrets Provider Job
```

Comment:
This app doesn't have anything to do with secrets provider.
Are we testing k8s?
The `Job`'s job is to update the k8s secrets.
I suggest to verify that instead.

Reply:
You are correct we dont need the helm-app because we are only concerned with updating the secret. I want to follow suit with the regression tests so lets keep this as is now